int-tests/src/test-integration/java/io/neow3j/compiler/DivisibleNFTIntegrationTest.java

```diff
@@ -62,6 +64,21 @@ public void testOwnerOf() throws IOException {
         assertThat(iter.get(1).getAddress(), is("NhxK1PEmijLVD6D4WSuPoUYJVk855L21ru"));
     }
 
+    @Test
+    public void testOwnerOfWithoutSessions() throws IOException {
```

Comment:
What happens when you try to get more items than available in the iterator?

Reply:
All items are returned. The cycle in the VM script is exited once the syscall `IteratorNext` returns false.

I'll add a test where the `maxIteratorReturnValues` is greater than the items in the iterator.